extensions/measurement-tracking/src/_shared/createReportAsync.js

```diff
@@ -19,17 +32,32 @@ async function createReportAsync(servicesManager, dataSource, measurements) {
     const naturalizedReport = await DICOMSR.storeMeasurements(
       measurements,
       dataSource,
-      ['ArrowAnnotate']
+      ['ArrowAnnotate'],
+      options
+    );
+
+    // The "Mode" route listens for DicomMetadataStore changes
+    // When a new instance is added, it listens and
+    // automatically calls makeDisplaySets
+    // We NEED the displaySetInstanceUID in this case, so we
+    // silence the DICOM Metadata Store's broadcast
+    const silent = true;
+    DicomMetadataStore.addInstances([naturalizedReport], silent);
```

Comment:
This seems like a hammer to a screw.

Can we just allow it to makeDisplaySets, and listen to displaySetAdded?

Reply:
Made this change.